src/app/component/header/header.component.html

```diff
@@ -112,21 +105,16 @@ <h1>Jyoti Technosoft LLP</h1>
     </div>
   </div>
 </div>
-<!-- TODO: Remove this and add inside dialog-service -->
```

Comment:
i have said to remove this model to modal service not to remove this comment

Reply:
ok  I have Add This comment